jenkins_jobs/modules/publishers.py

```diff
@@ -4639,7 +4639,7 @@ def robot(registry, xml_parent, data):
     for other_file in data.get("other-files", []):
         XML.SubElement(other_files, "string").text = str(other_file)
     XML.SubElement(parent, "disableArchiveOutput").text = str(
-        not data.get("archive-output-xml", True)
+        data.get("archive-output-xml", False)
```

Comment:
The XML tag represents the negation of the JJB `archive-output-xml` key, so lack of `not` is probably a mistake here.